src/cmd/cmd.cpp

```diff
@@ -62,31 +61,6 @@ static void nullMessageHandler(QtMsgType, const QMessageLogContext &, const QStr
 {
 }
 
-struct CmdOptions
```

Comment:
why removing it ?
that would have helped reduce the amount of modifications inside this file